src/main/web/src/scenes/Profile/ProfilePage.tsx

```diff
@@ -0,0 +1,80 @@
aVk3TnVaVE9hL2lHM2J0bDAyTmpaMkx4SGRFRmtCWVR3UUlPNEZDU3dHQVJBQUFSQUFnYVlUcUtxTStpTXA1ZjhoZk4rL1NtdTlub2krZ0J0dW04NGNEWUlBQ0lBQUNJQUFDUHd1RDRUTGY3eWR5V1FlRk1WaThTK01NZjhuRVgwQ0FnUXhBZ0lnQUFJZ0FBSWcwQ0lDNVZRUUlqb3VoUGp2WW5Cd2NKVzFOazlFc3lGQVdvUWN6WUlBQ0lBQUNJQ0Eyd1NpdCtXTzhma1hGaUIvWmEzbEFtUm5RWUM0SFIyd0hnUkFBQVJBQUFSYVJHQmNJcW9RNG45d0RzaWRRUkV5Q0pBV1VVZXpJQUFDSUFBQ0lBQUM1YUtvNVpNd1Vzb25lUVZraGJYMnYwYTJZSEQ2QlZFQ0FpQUFBaUFBQWlEUWJBTEJZWmp5S2R1LzV5VFVEY2FZZTdpWDRBNFkvaFVpcE5uWTBSNElnQUFJZ0FBSU9Fd2dlaFJYQ1BFZFVTZ1UvcGFJbGhDUkR1NkJnZjV3T0VCZ09naUFBQWlBQUFpMGlFQWxFVlVJOFU4c1FQNkJMNkVMYjhFTk9zVUtTSXZvbzFrUUFBRVFBQUVRY0pSQVJZQklLZmR6RHNqM3JiVi9FQ25ERHZIaGFHVEFiQkFBQVJBQUFSQm9JWUZvRHNnSExFRGV0TlplakNPNExVU09wa0VBQkVBQUJFQUFCS0piTUlZRnlQdlcybG1CQU1IcUJ3SUVCRUFBQkVBQUJFQ2dGUVRHQzVCQ29WRGlNN21SSEJDSWtGWmdSNXNnQUFJZ0FBSWdBQUxsY3V4Q2lQSUtpTGJXU3F5QUlDcEFBQVJBQUFSQUFBUmFUR0JTQWNMOVlnV2t4ZlRSUEFpQUFBaUFBQWc0U2dBcklJNDZIbWFEQUFpQUFBaUFRQ2NKUUlCMGtqNzZCZ0VRb2pqOUNRQUFCRUJKUkVGVUFBRVFBQUZIQ1VDQU9PcDRtQTBDSUFBQ0lBQUNuU1FBQWRKSit1Z2JCRUFBQkVBQUJCd2xBQUhpcU9OaE5naUFBQWlBQUFoMGtnQUVTQ2ZwbzI4UUFBRVFBQUVRY0pRQUJJaWpqb2ZaSUFBQ0lBQUNJTkJKQWhBZ25hU1B2a0VBQkVBQUJFREFVUUlRSUk0NkhtYURBQWlBQUFpQVFDY0pRSUIwa2o3NkJnRVFBQUVRQUFGSENVQ0FPT3A0bUEwQ0lBQUNJQUFDblNRQUFkSkordWdiQkVBQUJFQUFCQndsQUFIaXFPTmhOZ2lBQUFpQUFBaDBrZ0FFU0NmcG8yOFFBQUVRQUFFUWNKUUFCSWlqam9mWklBQUNJQUFDSU5CSkFoQWduYVNQdmtFQUJFQUFCRURBVVFJUUlJNDZIbWFEQUFpQUFBaUFRQ2NKUUlCMGtqNzZCZ0VRQUFFUUFBRkhDVUNBT09wNG1BMENJQUFDSUFBQ25TUUFBZEpKK3VnYkJFQUFCRUFBQkJ3bEFBSGlxT05oTmdpQUFBaUFBQWgwa2dBRVNDZnBvMjhRQUFFUUFBRVErUC9iczRQY2hLRWdpSUwvYy84ajJIZk1BUkltY2hJSktXdEVTM1N4Wk9GaGFqWlBwbFJBZ0pRZTN0b0VDQkFnUUNBcElFQ1MrbVlUSUVDQUFJRlNBUUZTZW5ockV5QkFnQUNCcElBQVNlcWJUWUFBQVFJRVNnVUVTT25oclUyQUFBRUNCSklDQWlTcGJ6WUJBZ1FJRUNnVkVDQ2xoN2MyQVFJRUNCQklDZ2lRcEw3WkJBZ1FJRUNnVkVDQWxCN2UyZ1FJRUNCQUlDa2dRSkw2WmhNZ1FJQUFnVklCQVZKNmVHc1RJRUNBQUlHa2dBQko2cHROZ0FBQkFnUktCUVJJNmVHdFRZQUFBUUlFa2dJQ0pLbHZOZ0VDQkFnUUtCVVFJS1dIdHpZQkFnUUlFRWdLQ0pDa3Z0a0VDQkFnUUtCVVFJQ1VIdDdhQkFnUUlFQWdLU0JBa3ZwbUV5QkFnQUNCVWdFQlVucDRheE1nUUlBQWdhU0FBRW5xbTAyQUFBRUNCRW9GQkVqcDRhMU5nQUFCQWdTU0FnSWtxVzgyQVFJRUNCQW9GUkFncFllM05nRUNCQWdRU0FvSWtLUysyUVFJRUNCQW9GUkFnSlFlM3RvRUNCQWdRQ0FwSUVDUyttWVRJRUNBQUlGU0FRRlNlbmhyRXlCQWdBQ0JwSUFBU2VxYlRZQUFBUUlFU2dVRVNPbmhyVTJBQUFFQ0JKSUNBaVNwYnpZQkFnUUlFQ2dWRUNDbGg3YzJBUUlFQ0JCSUNnaVFwTDdaQkFnUUlFQ2dWRUNBbEI3ZTJnUUlFQ0JBSUNrZ1FKTDZaaE1nUUlBQWdWSUJBVko2ZUdzVElFQ0FBSUdrZ0FCSjZwdE5nQUFCQWdSS0JRUkk2ZUd0VFlBQUFRSUVrZ0lDSktsdk5nRUNCQWdRS0JVUUlLV0h0ellCQWdRSUVFZ0tDSkNrdnRrRUNCQWdRS0JVUUlDVUh0N2FCQWdRSUVBZ0tTQkFrdnBtRXlCQWdBQ0JVZ0VCVW5wNGF4TWdRSUFBZ2FTQUFFbnFtMDJBQUFFQ0JFb0ZCRWpwNGExTmdBQUJBZ1NTQWdJa3FXODJBUUlFQ0JBb0ZSQWdwWWUzTmdFQ0JBZ1FTQW9Ja0tTKzJRUUlFQ0JBb0ZSQWdKUWUzdG9FQ0JBZ1FDQXA4QWlRNHppKzFscTN2ZmNrZjVIWkJBZ1FJRUNBd1BzTHpNeGVhOTMzZVo2Zk0zTzdWdDc3K3M2SEFBRUNCQWdRSVBCOGdabmZkeDE3NzU4QXVmL1Z5UE1uZVNJQkFnUUlFQ0JBNEovQTlhL0xGU0FmVjR6UUlVQ0FBQUVDQkFpOFNHQytBVmxHWkcxdDF6OVBBQUFBQUVsRlRrU3VRbUNDIi8+CjwvZGVmcz4KPC9zdmc+Cg==" // Verwende den tatsächlichen Pfad
```

Comment:
Use useState for the profile picture and fetch the image from ProfilePictureService

Reply:
the pr is only there to provide the user interface. The functionality comes with a later pr